src/main/java/net/hypixel/nerdbot/curator/ChannelGroupCurator.java

```diff
@@ -0,0 +1,321 @@
+    private final Emoji greenlit = NerdBotApp.getBot().getJDA().getEmojiById(NerdBotApp.getBot().getConfig().getEmojis().getGreenlit());
+
+    private final List<DiscordUser> users = new ArrayList<>();
+
+    public ChannelGroupCurator(boolean readOnly) {
+        super(readOnly);
+    }
+
+    @Override
+    public List<GreenlitMessage> curate(List<ChannelGroup> list) {
+        List<GreenlitMessage> output = new ArrayList<>();
+
+        setStartTime(System.currentTimeMillis());
+
+        for (ChannelGroup group : list) {
+            log("Starting to curate channel group " + group.getName() + " for guild " + group.getGuildId() + " (Source: " + group.getFrom() + ", Destination: " + group.getTo() + ")");
+
+            Guild guild = NerdBotApp.getBot().getJDA().getGuildById(group.getGuildId());
+            TextChannel suggestionChannel = NerdBotApp.getBot().getJDA().getTextChannelById(group.getFrom());
+            TextChannel submissionChannel = NerdBotApp.getBot().getJDA().getTextChannelById(group.getTo());
+            if (guild == null || suggestionChannel == null || submissionChannel == null) {
+                error("[" + group.getName() + "] Either the guild, suggestion channel, or submission channel is null so I can't continue with this group!");
+                continue;
+            }
+
+            List<Message> messages = suggestionChannel.getHistory().retrievePast(100).complete();
+            if (messages.isEmpty()) {
+                log("[" + group.getName() + "] No messages to curate in this group!");
+                continue;
+            }
+
+            log("[" + group.getName() + "] Found " + messages.size() + " messages to curate!");
+            int count = 0;
+
+            for (Message message : messages) {
+                log("[" + group.getName() + "] Curating message " + (++count) + "/" + messages.size() + " by " + message.getAuthor().getAsTag() + "!");
+
+                addEmojiIfMissing(group, message, agree);
+                addEmojiIfMissing(group, message, disagree);
+
+                MessageReaction positive = message.getReaction(agree);
+                MessageReaction negative = message.getReaction(disagree);
+                int realPositive = positive.getCount();
+                int realNegative = negative.getCount();
+
+                if (!Region.isDev()) {
+                    realPositive = discountBotAndUserReactions(group, message, positive);
+                    realNegative = discountBotAndUserReactions(group, message, negative);
+                }
+
+                if (realPositive == 0 && realNegative == 0) {
+                    log("[" + group.getName() + "] [" + message.getId() + "] Skipping because it has no positive and negative reactions!");
+                    continue;
+                }
+
+                if (realPositive < NerdBotApp.getBot().getConfig().getMinimumThreshold()) {
```

Comment:
make sure minimumThreshold is set

Reply:
It now has a default value if it's not set